docker/README.md

```diff
@@ -147,9 +147,3 @@ from the project checkout, and mounts it at `/opt/trino/gateway-ha-config.yml`.
 By default the container health check uses the file `docker/bin/health-check`
 mounted at `/usr/lib/trino/bin/health-check`. The scripts expects a 2XX response
 from the server at `/api/public/backends`.
-
```

Comment:
Could we link from this README to docs/development#Release_process.md in case someone looks for the information here?

Reply:
I think we dont want this for now .. this whole readme file will be moved into the docs soon anyway (and broken up)